### ISSUE TITLE
[Reopening #34] Crash on startup in a Forge Server

### ISSUE THREAD

**Issue by StuardBr** [OpenMods/OpenModsLib#35]
Reopening the issue #34, tested with the new snapshot and still crashing, but the error is another now.

Crash:http://pastebin.com/ESzxv4kZ

Versions:
OpenModsLib: 0.6-snapshot-295.jar and 0.6-snapshot-301.jar
Forge Version: 10.13.2.1263 

Thanks again for the atention :)

**Comment by boq** [comment]
Let me rephrase what I said in other thread: since this is similar to existing reports that were confirmed to be caused by user errors and I'm unable to reproduce this crash in clean Minecraft instance, it's very probable that problem is in your setup. Please try installing server from fresh in new folder and check if it works there.

**Comment by StuardBr** [comment]
Yes yes, i tried withou the other mods and the server starts ok. I will add one by one to see if it's a incopatibilty and i'll post here again to other people know what mod is causing this issue in OpenModslib

**Comment by boq** [comment]
Any explanations? Hints? Possible improvements?

**Comment by StuardBr** [comment]
Man, i installed all other mods, one by one and the problem is gone. I don't know what happen, but i think is like you said, an error in my side. Really, i don't know what happened.
